### ISSUE TITLE
Media title and channel have disappeared from attributes

### ISSUE THREAD

**Issue by 3raser95** [ollo69/ha-samsungtv-smart#175]
**Describe the bug**
I have an automation that checks whether I'm watching a specific channel and it used to work by checking {{ states.media_player.samsung_q60_series_65_2.attributes["media_title"] }} but it hasn't been working for the last month or so. It used to report the channel name but now it gives an error **UndefinedError: 'homeassistant.util.read_only_dict.ReadOnlyDict object' has no attribute 'media_title'**. Also in the developer tools in the attributes of the device there are no media_title or media_channel attributes that used to be there before.

**Expected behavior**
The media player device should report a media_title and media_channel attribute like it did before.

**Environment details:**
 - Environment (HASSIO, Raspbian, etc): Home Assistant OS on rpi3
 - Home Assistant version installed: 2022.7.6
 - Component version installed:  v0.7.4
 - Last know working version: Either 0.7.3 or 0.7.2 I'm not sure
 - TV model: 2019 Samsung Q60 series 65 inch

**Output of HA logs**
Paste the relavant output of the HA log here.

```
UndefinedError: 'homeassistant.util.read_only_dict.ReadOnlyDict object' has no attribute 'media_title'
```

**Comment by ollo69** [comment]
`media_title` attribute is there, probably you are not fetching in the right way. Did you check in `developer tools -> state` if you can see the attribute?

**Comment by 3raser95** [comment]
[Ey3scr34m](https://github.com/Ey3scr34m) actually mentioned a fix he found in #100 for this and I also found out that the TV had some sort of error in smart things where it was showing there but it reported as offline always even when the TV was on and connected to the internet. All the controls worked still through HA like changing channel and so forth but I'm guessing the smart things API wasn't working? I now fixed it by removing the TV from smart things and adding it back and now the "media_title" is working again.
